### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by kianmeng**
Found via `codespell -S *.pdf,framedmulticol.sty -L te,ist,bu,ot,nd,acn,noo,lod,fo,openin,inteval` and `typos --hidden --format brief`

### CODE REVIEW

**Comment by xkwxdyy** [comment]
There is one mistake in your PR: `\inteval` doesn't need to be modified, `\interval` does. Just a simple MWE to check
```latex
\documentclass{article}

\begin{document}

\inteval{1+3}

\end{document}
```
Be careful when you want to change a name of a macro.

**Comment by kianmeng** [comment]
@xkwxdyy Done. Please check again.

File: framedmulticol.sty
Comment (Sophanatprime) [inline framedmulticol.sty:1]: This copyright information is copied from [multicol](https://github.com/latex3/latex2e/blob/develop/required/tools/multicol.dtx) , so I prefer to keep it unchanged.

**Review by Sophanatprime — CHANGES_REQUESTED** [review]

**Comment by kianmeng** [comment]
:partying_face: :partying_face: :partying_face: :partying_face: :partying_face:

**Pull request merged**